lib/typeform_data/requestor.rb

```diff
@@ -11,6 +11,11 @@ module Requestor
 
       Errno::ECONNREFUSED,
       TypeformData::TransientResponseError,
+
+      # Sometimes it takes a while before Typeform's state becomes consistent. In particular, this
+      # can be an issue if you receive a webhook for a form response, then immediately request that
+      # response from Typeform's servers.
+      TypeformData::InvalidEndpointOrMissingResource
```

Comment:
This means retry on any 404 we get back from Typeform.

I don't feel strongly about whether to do this or not, but it feels a bit wrong to retry on any/all 404s we get back, especially as a workaround for a data consistency issue in their API implementation. If any consumer of this API relies on seeing 404s to detect absence of some resource in Typeform, it could make that code extemely slow.

Ideally, we'd handle that in some other way in our app code, using some knowledge that a response might not be present, but I understand if that's out of scope.

I'm okay with this as long as you update the docs to indicate that we retry on all 404s.